easybuild/easyconfigs/n/NWChem/NWChem-6.6.revision27746-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,40 @@
+name = 'NWChem'
+version = '6.6.revision27746'
+verdate = '2015-10-20'
+pyver = '2.7.11'
+versionsuffix = '-%s-Python-%s' % (verdate, pyver)
```

Comment:
Well, the version numbering is a bit problematic in case of `NWChem` but I am not able to find better way either. (For example how to indicate which patches are installed). `NWChem` is a bit rolling release, with snapshots (svn27746 = v6.6), and the patches are issued that affect old functionality, but patches are bringing new functionality will be released when a new snapshow will be released